doc/regression-tests.md

```diff
@@ -53,6 +53,12 @@ You *will* need to update it if you:
   * change the "questions & responses" file
   * change the routing logic
 
+Example:
+
+```shell
+bundle exec rails r script/generate-responses-and-expected-results-for-smart-answer.rb marriage-abroad
```

Comment:
it is hard for me to properly stand my ground because I understand you and have worked on this project for a while. A new developer will be better placed.